mmtbx/monomer_library/pdb_interpretation.py

```diff
@@ -2549,6 +2550,21 @@ def _get_next_residue():
           self._cif.cif["comp_specific_%s" % residue.resname.strip()] = mm.monomer.cif_object
         else:
           self._cif.cif["comp_%s" % residue.resname.strip()] = mm.monomer.cif_object
+      #
+      if mm.monomer is not None:
+        atom_dict = mm.monomer.atom_dict()
+        for i, atom in enumerate(residue.atoms()):
+          name = atom.name.strip()
+          if mm.mon_lib_names is not None:
+            name = mm.mon_lib_names[i]
+          al = atom_dict.get(name.strip(), None)
+          if al:
+            self.type_energies.append(al.type_energy)
+          else:
+            self.type_energies.append(None)
+            print(atom.quote())
+            assert 0
```

Comment:
Maybe put meaningful message in the assertion, like atom name so that when a user gets a crash here we'll get some more info straight in the bug-report.

Reply:
OK will change